MultimodalQnA/tests/test_compose_on_xeon.sh

```diff
@@ -20,8 +20,9 @@ export caption_fn="apple.txt"
 
 function build_docker_images() {
     cd $WORKPATH/docker_image_build
-    git clone https://github.com/opea-project/GenAIComps.git && cd GenAIComps && git checkout "${opea_branch:-"main"}" && cd ../
-
+    # TODO: Revert to the clone of opea-project "main" before merge
+    # git clone https://github.com/opea-project/GenAIComps.git && cd GenAIComps && git checkout "${opea_branch:-"main"}" && cd ../
+    git clone --single-branch --branch="mmqna-audio-query" https://github.com/mhbuehler/GenAIComps.git
     echo "Build all the images with --no-cache, check docker_image_build.log for details..."
     service_list="multimodalqna multimodalqna-ui embedding-multimodal-bridgetower embedding-multimodal retriever-multimodal-redis lvm-llava lvm-llava-svc dataprep-multimodal-redis"
```

Comment:
New images should be included in this service list, or it won't be build. If all the images in `build.yaml` required in this test, just remove the service_list, which means build all the images.

Reply:
Understood, thank you!